web/annotation_layer_builder.js

```diff
@@ -60,9 +60,11 @@ class AnnotationLayerBuilder {
   /**
    * @param {PageViewport} viewport
    * @param {string} intent (default value is 'display')
+   * @returns {Promise} A promise resolved when rendering the annotations is
+   * complete.
```

Comment:
It feels like this sentence is missing a couple of words; how about the following instead?
```suggestion
   * @returns {Promise} A promise that is resolved when rendering of the
   *   annotations is complete.
```